providers/hp/c7000/configure.go

```diff
@@ -575,8 +581,7 @@ func (c *C7000) User(users []*cfgresources.User) (err error) {
 				continue
 			}
 		}
-
-		if statusCode != 200 {
+		if statusCode != 200 && !userExists {
```

Comment:
Its ideal to use`http.StatusOK` const instead of `200` for status code comparison.
https://go.dev/src/net/http/status.go

Reply:
Done